sys/net/link_layer/Kconfig.lorawan

```
@@ -0,0 +1,483 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config HAVE_LORAWAN
+    bool
+    help
+        Indicates that LoRaWAN headers are present.
```

I'd change it to something like this
```suggestion
        Indicates that LoRaWAN support is present.
```

---

Makes sense.